workflow/rules/trim.smk

```diff
@@ -12,22 +12,33 @@ rule cutadapt_pe:
     log:
         "results/logs/cutadapt/{sample}-{unit}.log",
     wrapper:
-        "v1.22.0/bio/cutadapt/pe"
+        "v2.6.0/bio/cutadapt/pe"
 
 
-if not is_3prime_experiment:
+rule cutadapt:
```

Comment:
I have a doubt here, as mentioned in the `https://faqs.lexogen.com/faq/what-is-the-adapter-sequence-i-need-to-use-for-t-1` the reads are initially removed for polyA tails, then the adapters are removed in two separate Cutadapt steps. I am not sure if replacing it with a single command works as mentioned. Did you manage to check it with Quantseq reads?